examples/slide_puzzle/slide_puzzle.slint

```diff
@@ -215,6 +215,11 @@ export component MainWindow inherits Window {
     animate pieces-spacing { duration: 500ms; easing: ease-out; }
 
     Image {
+        // For the wasm build we want the puzzle to resize with the browser viewport, as per CSS in index.html.
```

Comment:
Note that this is not only for the wasm build, the default size of the window will also change with everything else.
But it is true that the otherwise default size doesn't matter, only in the case of the wasm code it does.